js/form.js

```diff
@@ -1,54 +1,20 @@
+const forms = document.querySelectorAll('form');
 const searchForm = document.querySelector('.ad-form');
-const serchFormInputs = searchForm.querySelectorAll('input');
-const serchFormSelects = searchForm.querySelectorAll('select');
-const searchFormTextArea = searchForm.querySelector('#description');
-
 const mapForm = document.querySelector('.map__filters');
-const mapFormInputs = mapForm.querySelectorAll('input');
-const mapFormSelects = mapForm.querySelectorAll('select');
-
-const buttonsInForm = document.querySelectorAll('button');
 
-const deactivateForm = () => {
-  searchForm.classList.add('.ad-form--disabled');
-  serchFormInputs.forEach((value) => {
-    value.disabled = true;
-  });
-  serchFormSelects.forEach((value) => {
-    value.disabled = true;
-  });
-  searchFormTextArea.disabled = true;
-  mapForm.classList.add('.map__filters--disabled');
-  mapFormInputs.forEach((value) => {
-    value.disabled = true;
-  });
-  mapFormSelects.forEach((value) => {
-    value.disabled = true;
-  });
-  buttonsInForm.forEach((value) => {
-    value.disabled = true;
+const toggleDisabledOnFormNodes = (isDisabled) => {
+  forms.forEach((value) => {
+    for (const element of value.elements) {
+      element.disabled = isDisabled;
+    }
   });
 };
```

Comment:
Лучше дизаблить только конкретные формы на странице. Вдруг завтра у нас появятся еще три формы ниже, которые отключать уже не требуется, тогда придется переписывать код. Также нейминг:

```suggestion
const toggleDisabledOnFormNodes = (isDisabled) => {
  [searchForm, mapForm].forEach((form) => {
    for (const element of form.elements) {
      element.disabled = isDisabled;
    }
  });
};
```